app/controllers/api/v1/meetings_controller.rb

```diff
@@ -81,25 +83,32 @@ def find_room
         @room = Room.find_by!(friendly_id: params[:friendly_id])
       end
 
-      def authorized_as_viewer?(access_code:)
-        return true if access_code.blank?
+      def authorized_as_viewer?(viewer_code:)
+        return true if viewer_code.blank? && params[:access_code].blank?
```

Comment:
We could also let the user type in anything in the Access Code Input, and he will be joining as a regular Viewer if the Access Code does not match the Moderator Access Code. I changed it because I felt it was bad UI but maybe it saves us a bunch of headaches.